### PR TITLE
Updated Readme and Documentation to improve grammar and english translation

### PR DESCRIPTION
**Pull request by wmthornton**
The Readme and Documentation files contained grammar errors which made them hard to read for native English speakers. Corrected the spelling and grammar mistakes and clarified sentences where I thought it was easier to read.

### CODE REVIEW

**Comment by Andrej123456789** [comment]
Thank you for creating Pull Request for ringOS

File: Documentation.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # DOCUMENTATION
-Documentation is for compiling, running and debugging ringOS.
+This documentation is intended to guide you in developing, compiling and running ringOS
```
Comment (Andrej123456789) [inline Documentation.md]: This guide also talk about debugging, put debugging backk

**Review by Andrej123456789 — CHANGES_REQUESTED** [review]

**Comment by wmthornton** [comment]
> Thank you for creating Pull Request for ringOS

You're welcome.

File: Documentation.md (hunk 1 above)
Comment (wmthornton) [reply Documentation.md]: I just updated the file to include "debugging"

File: Documentation.md [hunk 2]
```diff
@@ -34,31 +34,31 @@ If you first time running WSL or Linux installation, follow these steps:
...
 `cd kernel` or (if you are in gnu-efi folder) `cd ../kernel`
 
 `make kernel`
 
 ______________________________________________________________
 
-For running image,
+To run ringOS within Qemu on Linux or Windows:
 
-`make buildimg` [kernel folder]
```
Comment (Andrej123456789) [inline Documentation.md:55]: Where is make `make buildimg`

File: README.md [hunk 3]
```diff
@@ -15,16 +15,16 @@ ringOS source code
...
 ### Contributing and issues
-If you have problem or idea create **issue**.
+If you have problem or idea create an **issue**.
 
-If you want contribute in this project create **Pull Request**.
+If you want contribute to this project create a **Pull Request**.
 
 # About ringOS
-ringOS is ringwormGO operating system based on PonchoOS with supprot UEFI boot mode only.
+ringOS is ringwormGO operating system based on PonchoOS with support for UEFI boot modes only.
```
Comment (Andrej123456789) [inline README.md]: There is only BIOS or UEFI boot mode

**Review by Andrej123456789 — CHANGES_REQUESTED** [review]

File: Documentation.md (hunk 2 above)
Comment (wmthornton) [reply Documentation.md:55]: It was moved down under the Windows "run2.bat" instruction to keep the linux instructions together.

Comment (Andrej123456789) [reply Documentation.md:55]: I didn't see

File: README.md (hunk 3 above)
Comment (wmthornton) [reply README.md]: What I've written clarifies that - the language you used originally misspelled "support" and requires the adjective "for" to be gramatically correct. What I've written says the same thing as your original statement but is considered gramatically correct in english

Comment (Andrej123456789) [reply README.md]: That is correct, but stil there is one UEFI boot mode, we can use `modes` if we think on UEFI revisions

Comment (wmthornton) [reply README.md]: I removed the "s" in "modes" to make it singular

Comment (Andrej123456789) [reply README.md]: Good

File: Documentation.md (hunk 2 above)
Comment (Andrej123456789) [reply Documentation.md:55]: I saw it

**Comment by Andrej123456789** [comment]
Also, we have a [Discord server](https://discord.gg/zyzbdrDRQF) so you can ask there about ringOS

**Comment by Andrej123456789** [comment]
I will merge this Pull Request now

**Pull request merged**